packages/lesswrong/components/tagging/TagPageTitle.tsx

```diff
@@ -22,8 +24,8 @@ const TagPageTitle = ({isSubtitle, classes}: {
     return null;
   } else {
     return <Helmet>
-      <title>{`${tag.name} tag`}</title>
-      <meta property='og:title' content={`Posts tagged ${tag.name}`}/>
+      <title>{titleString}l</title>
```

Comment:
This probably isn't supposed to have an l?

Reply:
Yep, good catch